src/writer/pdb-writer.ts

```diff
@@ -118,7 +118,7 @@ export default class PdbWriter extends Writer {
           defaults(a.element, '')
         ))
         ia += 1
-      })
+      }, this.structure.getSelection() || undefined)
```

Comment:
Would it make sense to have `getSelection()` default to `undefined` rather than `false` to simplify this?
This looks great btw!

Reply:
Yes :) Fixed in another commit